build_tools/fetch_sources.py

```diff
@@ -274,6 +274,7 @@ def main(argv):
         type=str,
         default=[
             "MIOpen",
+            "composable_kernel",
```

Comment:
Given that
1. We don't yet want to use CK on Windows
2. CK even fails to checkout on Windows due to long file paths: https://github.com/ROCm/TheRock/actions/runs/15766388716/job/44443647119?pr=876#step:9:163 `error: unable to create file library/src/tensor_operation_instance/gpu/gemm_multiply_multiply/device_gemm_multiply_multiply_xdl_f8_f8_bf16/device_gemm_multiply_multiply_xdl_f8_f8_bf16_mk_nk_mn_comp_mfma16x16_default_instance_part1.cpp: Filename too long`

We can make its inclusion here conditional on is_windows (see elsewhere in the file). That way, Windows users and CI will not even try to pull down the submodule unless they manually run git commands